src/fields/field_algo_base.impl.hh

```diff
@@ -47,7 +47,7 @@ template <int spacedim, class Value>
 FieldAlgorithmBase<spacedim, Value>::FieldAlgorithmBase(unsigned int n_comp)
 : value_(r_value_),
   field_result_(result_other),
-  component_idx_(std::numeric_limits<unsigned int>::max()),
+  component_idx_(0),
```

Comment:
Point is that it should be set to undef_idx.

Reply:
done